New Game Overlays and PP Counter

Ayreth opened the pull request:
Added:
  - Bon Toutou Game Overlay
  - Tiens un Os PP Counter
  - Donne la patte Game Overlay

Ayreth committed:
Add files via upload

l3lackShark commented:
Thanks for your contribution! Our Front-Ender will review your code within the next couple of days.

Ayreth committed:
Update README.md

Ayreth committed:
1

2

Ayreth committed:
2

3

Dartandr commented:
Hello
1) Please change folders names from "-+-+ Bon Toutou Game Overlay +-+-" to BonToutouGameOverlay and same with others it's kinda a standard for us 
2) Screenshot of "Bon Toutou" has different hpbar than it is in reality (change phbar in code or change a screenshot)
3) I didn't check code yet but "Donne la Patte" just doesn't work

Ayreth committed:
Rename -+-+ Bon Toutou Game Overlay +-+-/css/odometr.css to BonToutTouGameOverlay/css/odometr.css

Ayreth committed:
Revert "Rename -+-+ Bon Toutou Game Overlay +-+-/css/odometr.css to BonToutTouGameOverlay/css/odometr.css"

This reverts commit e3539d7f4125b420ccf98a9f3e18c7c17e96d3d2.

Ayreth committed:
Applied Changes

Renamed the folders + changed DonneLaPatte to DonneLaPapatte
Changed BouToutou's screenshots
Changed the DonneLaPapatte/DonneLaPatte's code so it works

Ayreth committed:
Slight fix to README.MD

Ayreth commented:
Renamed the folders and changed DonneLaPapatte's Code

Dartandr commented:
1) hp bar doesn't work neither in "Bon Toutou Game Overlay" nor in "Donne la patte Game Overlay"
2) you accidentally added two folders of this counter in your repo 
![image](https://user-images.githubusercontent.com/32641958/100863561-c9833700-34a5-11eb-8431-7ef47d72aeae.png)

3) you probably need to add default "SS" value for score rank cause it's strange that rank is D before map is even started 
![image](https://user-images.githubusercontent.com/32641958/100862832-ba4fb980-34a4-11eb-8053-fdd61fb13a36.png)
4) at the and of map on counter transition silver S some how changed to regular S

Ayreth committed:
Wiilk

Changed :
   -  Removed DonneLaPatte
   -  Changed the default value from D to SS
   - Update the ranking panel's code from gameplay to ResultScreen
   - Added 3 dots if the Leaderboard's name is too long
   - Slight fixes

Ayreth committed:
Update style.css

Ayreth committed:
Update style.css

Ayreth commented:
1. I tried to see where the problem would come from, ive sent it to 10 different people and none of them had this bug, However i still changed it from being a local file to an host file.

2. Removed.

3. Fixed

4. Comes from the fact that you watch a replay without any mods while it has some. If you enable HD or FL before watching one, it'll be changed to it. Nothing i can do on my end to fix it

Dartandr commented:
for now there is no hp bar at all https://cdn.discordapp.com/attachments/530940222771560452/783790643314950214/2020-12-02_23-20-32.mp4

ToastedDev commented:
Has this been inactive for 26 days? Or is it just me?
